docs/studies/covid-symptom.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Covid Symptom
+parent: Studies
+grand_parent: ETL
+nav_order: 1
+# audience: engineer familiar with the project
+# type: howto
+---
+
+# The Covid Symptom Study
```

Comment:
I figured if the docker compose has study specific commands now, we might want a per-study doc about how to run the study or other study-specific notes.